src/components/InputSwitch/index.tsx

```diff
@@ -25,12 +25,24 @@ interface InputSwitchProps {
 }
 
 const InputSwitch = (props: InputSwitchProps): JSX.Element => {
-    const { displayName, inputType, dataType, description, min, max, options, id, gradientOptions, conversionFactor, unit } = props;
+    const {
+        displayName,
+        inputType,
+        dataType,
+        description,
+        min,
+        max,
+        options,
+        id,
+        gradientOptions,
+        conversionFactor,
+        unit
+    } = props;
 
     const selectedRecipeId = useSelectedRecipeId();
-    const updateRecipeObj = useUpdateRecipeObj();
+    const editRecipe = useEditRecipe();
     const getCurrentValue = useGetCurrentValue();
-    const recipeVersion = useCurrentRecipeString();
+    const recipeVersion = useRecipes();
```

Comment:
this is a little confusing to me. I expect the recipeVersion to be a field in the recipe. and useRecipes() to give you every available recipe